TombEngine/Sound/sound.cpp

```diff
@@ -1056,26 +1070,27 @@ int GetShatterSound(int shatterID)
 		return SFX_TR4_SMASH_ROCK;
 }
 
+// TODO: Revise flags.
```

Comment:
Which flags you want to revise?

Reply:
I think this was a note for myself to make a flag enum.